tardis/apps/publication_workflow/static/publication-tokens/publication_tokens.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Tokens controller
+ *
+ */
+/*eslint no-unused-vars: ["error", { "argsIgnorePattern": "^_" }]*/
+angular
+.module('publicationWorkflow')
+.controller('PublicationTokensController', function (ngDialog, $scope, $log, $http, $resource) {
+    var vm = this;
+    vm.tokensListRes = $resource('/apps/publication-workflow/tokens_json/:publication_id/', {}, {
+        'get': {method: 'GET', isArray: true}
+    });
+    if (angular.isDefined($scope.$parent.ngDialogData)) {
+        vm.publicationId = $scope.$parent.ngDialogData.publicationId;
+    }
+    else {
+        throw new Error("publicationId is undefined in PublicationTokensController");
+    }
+
+    vm.tokensListRes.get({'publication_id': vm.publicationId}).$promise.then(function (data) {
+        vm.tokens = data;
+    });
+
+    // Create a token and open tokens dialog
+    vm.createToken = function () {
```

Comment:
Codacy has a fix for the issue: [Unexpected space before function parentheses.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933947)
```suggestion
    vm.createToken = function() {
```